test/fielddef.test.ts

```diff
@@ -152,11 +152,11 @@ describe('fieldDef', () => {
 
   describe('title()', () => {
     it('should return correct title for aggregate', () => {
-      assert.equal(title({field: 'f', type: QUANTITATIVE, aggregate: 'mean'}, {}), 'Mean of f');
+      assert.equal(title({field: 'f', aggregate: 'mean'}, {}), 'Mean of f');
     });
 
     it('should return correct title for count', () => {
-      assert.equal(title({field: '*', type: QUANTITATIVE, aggregate: 'count'}, {countTitle: 'baz!'}), 'baz!');
+      assert.equal(title({field: '*', aggregate: 'count'}, {countTitle: 'baz!'}), 'baz!');
```

Comment:
remove field: *

Reply:
Let's do that in another PR to clean up all of them.